deployment/utils.py

```diff
@@ -156,3 +158,33 @@ def registry_filepath_from_domain(domain: str) -> Path:
         raise ValueError(f"No registry found for domain '{domain}'")
 
     return p
+
+
+def sample_nodes(
+        domain: str,
+        num_nodes: int,
+        random_seed: Optional[int] = None,
+        duration: Optional[int] = None
+):
+    porter_endpoints = {
+        MAINNET: "https://porter.nucypher.community/bucket_sampling",
+        LYNX: "https://porter-lynx.nucypher.network/get_ursulas",
+        TAPIR: "https://porter-tapir.nucypher.network/get_ursulas",
+    }
```

Comment:
You could make this dictionary a constant within the file itself instead of in the function.

Reply:
✔️